vendor/fontawesome/scss/_variables.scss

```diff
@@ -0,0 +1,1468 @@
+// Variables
+// --------------------------
+
+$fa-font-path: "../-/static" !default;
```

Comment:
Instead of changing the path in the vendored file (which would override it when we update FA), could you set the variable in our own `base.scss`?

Reply:
This was the recommendation in [fontawesome's docs](https://fontawesome.com/how-to-use/on-the-web/using-with/sass#compile), but I can do whatever

Reply:
Why are they recommending that :/

Reply:
I have no idea to be perfectly honest

Reply:
Note that this will be fixed whenever we can use [scss modules](https://sass-lang.com/documentation/at-rules/use), at that time we can use something along these lines
```scss
@use "fontawesome" with (
    $fa-font-path: "../-/static"
);
// For more info see https://sass-lang.com/documentation/at-rules/use#configuration
```
Modules are nice for other reasons as well, they allow namespacing and explicit access to external variables, mixins and rules